pkg/driver/node.go

```diff
@@ -121,6 +122,15 @@ func (d *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return nil, status.Error(codes.InvalidArgument, "Volume capability not supported")
 	}
 
+	var pv *corev1.PersistentVolume
+	var err error
+	if d.k8sClient != nil {
+		pv, err = d.k8sClient.GetPersistentVolume(ctx, volumeID)
+		if k8serrors.IsNotFound(err) {
+			klog.Warning("pv volumeHandle is not equal to pv name, may be duplicated resulting in unexpectd issues")
```

Comment:
add related vars into log text

```suggestion
			klog.Warning("volumeHandle %s is not the same as pv name %s, mount pod might not be created", ??, ??)
```

Reply:
If it's not equal, we can't get the name of the PV in publish phase

Reply:
ok, then just print what we got